conf/etc/v2/cli_schema.json

```diff
@@ -0,0 +1,121 @@
+{
+    "version": "2.0.0",
+    "cluster": {
+        "help": {
+            "usage": "cortx cluster [-h | --help]",
+            "class": "ha.cli.exec.commandExecutor.ClusterCLIUsage"
+        },
+        "start": {
+            "usage": "cortx cluster start [-h] [--all | --server] [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterStartExecutor"
+        },
+        "stop": {
+            "usage": "cortx cluster stop [-h] [--all | --server] [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterStopExecutor"
+        },
+        "restart": {
+            "usage": "cortx cluster restart",
+            "class": "ha.cli.exec.clusterExecutor.ClusterRestartExecutor"
+        },
+        "standby": {
+            "usage": "cortx cluster standby [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterStandbyExecutor"
+        },
+        "active": {
+            "usage": "cortx cluster active [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterActiveExecutor"
+        },
+        "list": {
+            "usage": "cortx cluster list [nodes|storagesets|services]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterListExecutor"
+        },
+        "status": {
+            "usage": "cortx cluster status [all|hw|services] [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterStatusExecutor"
+        },
+        "add_node": {
+            "usage": "cortx cluster add_node [--nodeid <node id> | --descfile <node description file>] [ --username <cluster_username> ] [ --password <cluster_password> ] [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterNodeAddExecutor"
+        },
+        "add_storageset": {
+            "usage": "cortx cluster add_storageset [<storageset id>] [ --descfile <storageset description file>] [--json]",
+            "class": "ha.cli.exec.clusterExecutor.ClusterNodeAddExecutor"
```

Comment:
ClusterNodeAddExecutor?

Reply:
Removed add_storageset as its Executer not yet implemented